src/components/icons/StartIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+/**
+ * An icon to display on the start page of a service.
+ */
+export const StartIcon: React.FunctionComponent = (): JSX.Element => (
+  <svg
+    className="govuk-button__start-icon"
```

Comment:
We should be passing this class in, not using it here directly, as we're not importing the stylesheet that might reference it.